pkg/neorpc/filters.go

```diff
@@ -48,6 +51,43 @@ type (
 		Type   *mempoolevent.Type `json:"type,omitempty"`
 	}
 )
+type ValidFilter interface {
+	IsValid() error
+}
+
+// IsValid checks whether the filter is valid.
+func (f *BlockFilter) IsValid() error {
+	return nil
```

Comment:
You can move this comment to `NotificationFilter` interface, and here add just a `IsValid implements NotificationFilter interface.` note. Usually it's done in order not to change every single implementation documentation in case if something is changed in the logic description.